Makefile

```diff
@@ -16,11 +31,18 @@ clean-pyc: ## remove Python file artifacts
 	find . -name '*~' -exec rm -f {} +
 	find . -name '__pycache__' -exec rm -fr {} +
 
+clean-state: ## remove the build state
+	rm -rf $(STATEDIR)
+
 clean-test: ## remove test and coverage artifacts
 	rm -fr .tox/
 	rm -fr assets/
 	rm -f report.html
 
+test: cnx_slim_dump.sql.gz $(STATEDIR)/docker-build
```

Comment:
Would it be a terrible idea to add test commands for each of the markers? Maybe this can be handled via environment variable? Something like:
```
test_webview: cnx_slim_dump.sql.gz $(STATEDIR)/docker-build
	docker-compose up -d selenium-chrome
	docker-compose exec selenium-chrome wait-for db:5432 -- tox -- -m webview --webview_base_url=http://ui:8000 --archive_base_url=http://archive:6543
```